aws/files/userdata_agent

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash -x
+export curlprefix=appropriate
```

Comment:
curlprefix is only required if using RancherOS

Reply:
There are parts of the userdata_agent script that utilize $curlprefix to pull the "appropriate/curl" image, should we hardcode that to "appropriate" instead?